classtime.js

```diff
@@ -1,210 +1,321 @@
+                    }
                 },
                 {
-                    name: "Lunch",
-                    startTime: {hours: 11, minutes:30},
-                    endTime: {hours: 12, minutes:05}
+                    "name": "Lunch",
+                    "startTime": {
+                        "hours": 11, "minutes": 30
+                    },
+                    "endTime": {
+                        "hours": 12, "minutes": 05
+                    }
                 },
                 {
-                    name: "Passing Period",
-                    showInFullSchedule: false,
-                    startTime: {hours: 12, minutes:05},
-                    endTime: {hours: 12, minutes:10}
+                    "name": "Passing Period",
+                    "showInFullSchedule": false,
+                    "startTime": {
+                        "hours": 12, "minutes": 05
+                    },
+                    "endTime": {
+                        "hours": 12, "minutes": 10
+                    }
                 },
                 {
-                    name: "3rd Period",
-                    startTime: {hours: 12, minutes:10},
-                    endTime: {hours: 13, minutes:35}
+                    "name": "3rd Period",
+                    "startTime": {
+                        "hours": 12, "minutes": 10
+                    },
+                    "endTime": {
+                        "hours": 13, "minutes": 35
+                    }
                 },
                 {
-                    name: "Passing Period",
-                    showInFullSchedule: false,
-                    startTime: {hours: 13, minutes:35},
-                    endTime: {hours: 13, minutes:40}
+                    "name": "Passing Period",
+                    "showInFullSchedule": false,
+                    "startTime": {
+                        "hours": 13, "minutes": 35
+                    },
+                    "endTime": {
+                        "hours": 13, "minutes": 40
+                    }
                 },
                 {
-                    name: "4th Period",
-                    startTime: {hours: 13, minutes:40},
-                    endTime: {hours: 15, minutes:05}
+                    "name": "4th Period",
+                    "startTime": {
+                        "hours": 13, "minutes": 40
+                    },
+                    "endTime": {
+                        "hours": 15, "minutes": 05
+                    }
                 }
             ]
         }
     ]
-};
+}
 
-// settings: {
-//         militaryTime: true;
-//     }
+var currentDay = 0;
+var currentHours = 0;
+var currentMinutes = 0; 
+var currentSeconds = 0;
+var currentDate;
 
+var currentClassPeriodIndex = -1;
+var currentScheduleIndex = -1;
+var use24HourTime = getLocalStorageBoolean("use24HourTime", false);
 
 function update() {
     updateTime();
-    if (typeof data !== 'undefined') {
```

Comment:
why was this removed? it is supposed to check if the data exists (mostly for once it is moved to a database)